packages/hdwallet-tally/src/ethereum.ts

```diff
@@ -0,0 +1,130 @@
+import * as core from "@shapeshiftoss/hdwallet-core";
+
+export function describeETHPath(path: core.BIP32Path): core.PathDescription {
+  const pathStr = core.addressNListToBIP32(path);
+  const unknown: core.PathDescription = {
+    verbose: pathStr,
+    coin: "Ethereum",
+    isKnown: false,
+  };
+
+  if (path.length !== 5) return unknown;
+
+  if (path[0] !== 0x80000000 + 44) return unknown;
+
+  if (path[1] !== 0x80000000 + core.slip44ByCoin("Ethereum")) return unknown;
+
+  if ((path[2] & 0x80000000) >>> 0 !== 0x80000000) return unknown;
+
+  if (path[3] !== 0) return unknown;
+
+  if (path[4] !== 0) return unknown;
+
+  const index = path[2] & 0x7fffffff;
+  return {
+    verbose: `Ethereum Account #${index}`,
+    accountIdx: index,
+    wholeAccount: true,
+    coin: "Ethereum",
+    isKnown: true,
+  };
+}
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+export async function ethVerifyMessage(msg: core.ETHVerifyMessage, ethereum: any): Promise<boolean | null> {
+  console.error("Method ethVerifyMessage unsupported for Tally wallet!");
+  return null;
+}
+
+export function ethGetAccountPaths(msg: core.ETHGetAccountPath): Array<core.ETHAccountPath> {
+  const slip44 = core.slip44ByCoin(msg.coin);
+  if (slip44 === undefined) return [];
+  return [
+    {
+      addressNList: [0x80000000 + 44, 0x80000000 + slip44, 0x80000000 + msg.accountIdx, 0, 0],
+      hardenedPath: [0x80000000 + 44, 0x80000000 + slip44, 0x80000000 + msg.accountIdx],
+      relPath: [0, 0],
+      description: "Tally",
+    },
+  ];
+}
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+export async function ethSignTx(msg: core.ETHSignTx, ethereum: any, from: string): Promise<core.ETHSignedTx | null> {
+  console.error("Method ethSignTx unsupported for Tally wallet!");
+  return null;
+}
+
+export async function ethSendTx(msg: core.ETHSignTx, ethereum: any, from: string): Promise<core.ETHTxHash | null> {
+  try {
+    const utxBase = {
+      from: from,
+      to: msg.to,
+      value: msg.value,
+      data: msg.data,
+      chainId: msg.chainId,
+      nonce: msg.nonce,
+      // Tally, like other Web3 libraries, derives its transaction schema from Ethereum's official JSON-RPC API specification
+      // (https://github.com/ethereum/execution-apis/blob/d63d2a02bcd2a8cef54ae2fc5bbff8b4fac944eb/src/schemas/transaction.json).
+      // That schema defines the use of the label `gas` to set the transaction's gas limit and not `gasLimit` as used in other
+      // libraries and as stated in the official yellow paper (https://ethereum.github.io/yellowpaper/paper.pdf).
+      gas: msg.gasLimit,
+    };
+
+    const utx = msg.maxFeePerGas
```

Comment:
We don't need this ternary operator if Tally does not support EIP-1559.

Reply:
It does support EIP-1559, that was my bad and I have rectified the discrepancy above.